Build/libHttpClient.Apple.C/libHttpClient.xcodeproj/project.pbxproj

```diff
@@ -106,6 +180,27 @@
 			remoteGlobalIDString = 67EE817D2362599C00A5F3DE;
 			remoteInfo = SSL;
```

Comment:
>SSL [](start = 16, length = 3)

Looks like you didn't rename the original SSL to SSL_iOS. Intentional?

Reply:
That's so weird. It shows in the UI as `SSL_iOS`, so I'm pretty unclear what this refers to...